### PR TITLE
[DOC] Update CONTRIBUTING.md for feature flags

### PR DESCRIPTION
**Pull request by sbatson5**
Addresses the latest comment on https://github.com/emberjs/data/issues/4238#issuecomment-198482078 for adding feature-flags for new features

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,6 +58,54 @@ this will also help us to understand it better ourselves.
...
+  if (isEnabled('ds-mynew-feature')) {
+    test('test for new feature', function(assert) {
+      // ...
+    })
+  }
+  ```
+
+  This will allow the test suite to run as normal.
+
+4. Running tests with all feature flags enabled is possible via `ember test --environment=test-optional-features`
```
Comment (pangratz) [inline CONTRIBUTING.md]: Can you add a note that this is also possible in the browser via the `Enable Opt Features` checkbox. This is handy if you need to debug stuff which is behind a feature flag.

**Comment by pangratz** [comment]
Awesome @sbatson5, thanks for tackling this! :heart: 

I left one small comment, after that this should be good to go! :rocket:

**Comment by sbatson5** [comment]
Thanks @pangratz!  Added info about the `Enable Opt Feature` checkbox.

**Comment by pangratz** [comment]
Thank you kindly @sbatson5! Welcome to the list of `ember-data` contributors! :tada: :heart:

**Pull request merged**